src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/Internal/Win32.cs

```diff
@@ -20,7 +20,7 @@ internal static class Win32
                 snapshotHandle = Interop.Kernel32.CreateToolhelp32Snapshot(Interop.Kernel32.SnapshotFlags.Process, 0);
 
                 Interop.Kernel32.PROCESSENTRY32 procEntry = default(Interop.Kernel32.PROCESSENTRY32);
-                procEntry.dwSize = Marshal.SizeOf(typeof(Interop.Kernel32.PROCESSENTRY32));
+                procEntry.dwSize = Marshal.SizeOf<Interop.Kernel32.PROCESSENTRY32>();
```

Comment:
```suggestion
                procEntry.dwSize = (int)sizeof(Interop.Kernel32.PROCESSENTRY32);
```
PROCESSENTRY32 is fully blittable structs. No need to use legacy marshalling APIs.

Reply:
Would it make sense to update this message?

https://github.com/dotnet/runtime/blob/e71a9583b4d6c9bd97edd87cda7f98f232f63530/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs#L113

Reply:
Fixed.